virtualize.sh

```diff
@@ -148,6 +148,12 @@ ssh $SSHOPTS root@$installserverip "
 
 kill ${tail_job}
 
+# Dump elasticsearch logs into /var/log/elasticdump,
+# upload_logs will update the dump in swift.
+scp $SSHOPTS $ORIG/dumpelastic.py root@$installserverip:/tmp
```

Comment:
Do you think it's possible to adjust `dumpelastic.py` in order to be able to call it directly from the local machine? For example, by adding a `--target-server=$installserverip argument`.